apps/patient/src/views/Status.tsx

```diff
@@ -45,54 +39,53 @@ export const Status = () => {
       order?.fulfillment?.type !== types.FulfillmentType.MailOrder
   );
 
-  const [error, setError] = useState(undefined);
   const [submitting, setSubmitting] = useState<boolean>(false);
   const [successfullySubmitted, setSuccessfullySubmitted] = useState<boolean>(false);
+  const [enrichedPharmacy, setEnrichedPharmacy] = useState<EnrichedPharmacy | undefined>(undefined);
 
   const { fulfillment, pharmacy, organization, address } = order;
 
   const fulfillmentType = getFulfillmentType(pharmacy?.id, fulfillment, type);
 
   const toast = useToast();
 
-  const markOrderAsPickedUp = async () => {
-    try {
-      setSubmitting(true);
+  const handleMarkOrderAsPickedUp = async () => {
+    setSubmitting(true);
 
-      graphQLClient.setHeader('x-photon-auth', token);
-      const results: any = await graphQLClient.request(MARK_ORDER_AS_PICKED_UP, {
-        markOrderAsPickedUpId: orderId
-      });
+    try {
+      const result: boolean = await markOrderAsPickedUp(orderId, token);
 
       setTimeout(() => {
```

Comment:
I know this not new, but I never noticed this timeout... what's it for?

Reply:
smoothing out the flow, instant resolution is jumpy